pkgs/http_client_conformance_tests/lib/src/redirect_tests.dart

```diff
@@ -51,9 +51,7 @@ void testRedirect(Client client, {bool redirectAlwaysAllowed = false}) async {
           client.send(request),
           throwsA(isA<ClientException>()
               .having((e) => e.message, 'message', 'Redirect limit exceeded')));
-    },
-        skip: 'Re-enable after https://github.com/dart-lang/sdk/issues/49012 '
```

Comment:
This should probably be:
`redirectAlwaysAllowed ? 'redirects always allowed' : false`